README.md

```diff
@@ -158,9 +139,6 @@ An opinioned coding guideline for React + Redux project @WORD UP
     * Since we're in a SPA, remember to remove the listener at a proper timing like `componentDidUnmount`.
     * Take the parent/child tree into account. Don't add the same listener at both parents and children.
 
-## Under consideration (welcome ideas as well as debates)
```

Comment:
I think group actions by category will be better than group by sync/async

Reply:
I got this idea from this post. The benefit is easier to write tests.
https://tech.affirm.com/redux-patterns-and-anti-patterns-7d80ef3d53bc

Actually, I haven't really tried this out, don't know how it feels. But I feel Jest makes async test also quite easy(so we don't really need to separate them out)? Do you have any idea about this?

Reply:
1. I agree we should create action creator for sync action, but no need to split to two files, I think this will make us harder to file the code.
2. we use promise middleware, so we can't follow this rule base on the current implementation.

Reply:
ok, agree.